src/components/compose.jsx

```diff
@@ -604,7 +604,15 @@ function Compose({
         const item = items[i];
         if (item.kind === 'file') {
           const file = item.getAsFile();
-          if (file && supportedMimeTypes.includes(file.type)) {
+          if (!file) {
+            alert(`Could not access the given attachment.`);
```

Comment:
Want to do this so the error gets translated?

```.js
alert(t`Could not access the given attachment.`);
```

(and same for the second error.)

disclaimer: I'm not Chee Aun and have no authority over whether this gets merged, just an observation I hope is helpful

Reply:
```suggestion
            alert(t`Could not access the given attachment.`);
```

Like this?